common/templates/context_funcs.go

```diff
@@ -23,7 +23,8 @@ import (
 
 var (
 	ErrTooManyCalls    = errors.New("too many calls to this function")
-	ErrTooManyAPICalls = errors.New("too many potential discord api calls function")
+	ErrTooManyAPICalls = errors.New("too many potential Discord API calls")
+	ErrRegexCacheLimit = errors.New("too many unique RegExes")
```

Comment:
call it "too many regular expressions ( regex)  used" and end all the unnecessary debate.

Reply:
I don't mean to continue it but I believe "unique" is a key work here and should be kept. Else it could be taken as too many regex function calls, and not just unique regular expressions.

Reply:
this is pondering over nothing, just give it a meaningful description and end it.

Reply:
if people complain, raise another pr to improve it.